starknet_py/tests/e2e/fixtures/misc.py

```diff
@@ -94,6 +99,47 @@ def get_tx_status_full_node_client():
     return f"{FullNodeClient.__module__}.FullNodeClient.get_transaction_status"
 
 
+class ContractVersion(Enum):
+    V1 = "V1"
+    V2 = "V2"
+
+
+class UnkonwnArtifacts(BaseException):
+    pass
+
+
+def load_contract(contract_name: str, version: Optional[ContractVersion] = None):
+    if version is None:
+        if "--contract_dir=v1" in sys.argv:
+            version = ContractVersion.V1
+        if "--contract_dir=v2" in sys.argv:
+            version = ContractVersion.V2
+
+    artifacts_path = CONTRACTS_V2_ARTIFACTS
+    directory = CONTRACTS_V2_COMPILED
+
+    if version == ContractVersion.V1:
+        artifacts_path = CONTRACTS_V1_ARTIFACTS
+        directory = CONTRACTS_V1_COMPILED
+
+    loaded_artifcats = json.loads((artifacts_path).read_text("utf-8"))
+
+    for item in loaded_artifcats["contracts"]:
+        if item["contract_name"] == contract_name:
+            artifacts = item["artifacts"]
+            break
+
+    if not isinstance(artifacts, dict):  # pyright: ignore
+        raise UnkonwnArtifacts(
+            f"Artifacts for contract {contract_name} haven't be founded"
+        )
+
+    sierra = (directory / artifacts["sierra"]).read_text("utf-8")
+    casm = (directory / artifacts["casm"]).read_text("utf-8")
+
+    return {"casm": casm, "sierra": sierra}
+
+
 def read_contract(file_name: str, *, directory: Optional[Path] = None) -> str:
```

Comment:
Can we somehow combine `load_contract` and `read_contract` to have only one function for this?

Reply:
I simplified `read_contract` and left both functions, let me know, what do you think